Packs/CheckpointFirewall/Integrations/CheckPointFirewallV2/CheckPointFirewallV2.py

```diff
@@ -1641,6 +1656,60 @@ def checkpoint_show_task_command(client: Client, task_id: str) -> CommandResults
     return command_results
 
 
+def checkpoint_add_objects_batch_command(client: Client, object_type: str, ipaddress, name):
+    printable_result = {}
```

Comment:
```suggestion
    context_data = {}
```

Reply:
please explain more how to use context_data ={}

Reply:
It's simply a new name, you just need to use that instead of `printable_result`